test/basic.html

```diff
@@ -354,13 +354,19 @@
           expect(customField.hasAttribute('has-value')).to.be.false;
         });
 
-        it('should not have has-value on custom field if only date or time is selected', () => {
+        it('should not have has-value on custom field if only time is selected', () => {
           timePicker.value = '15:00';
           dateTimePicker.__triggerCustomFieldValueUpdate();
           expect(customField.hasAttribute('has-value')).to.be.false;
         });
 
-        it('should have has-value on custom field if both date or time are selected', () => {
+        it('should not have has-value on custom field if only date is selected', () => {
+          datePicker.value = '15:00';
```

Comment:
This is not a valid value for date picker.

Reply:
Yeah 😅 
Updated.